### PR TITLE
Mid Term blog of OSPP 2021: Wechaty Piggy Bro

### PR DESCRIPTION
**Pull request by ReiKohaku**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  
- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

### CODE REVIEW

File: jekyll/_posts/2021-08-26-ospp-mid-term-wechaty-piggy-bro.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  - chat
+  - summer-of-wechaty
+  - summer-2021、
+  - ecosystem
+  - ospp
+  - ospp-2021
+  - mid-term
+---
+
+> Author: [@ReiKohaku](https://github.com/ReiKohaku)
```
Comment (lijiarui) [inline jekyll/_posts/2021-08-26-ospp-mid-term-wechaty-piggy-bro.md]: Please remove Author's description here.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please get your mentor approve first.

File: jekyll/_posts/2021-08-26-ospp-mid-term-wechaty-piggy-bro.md (hunk 1 above)
Comment (ReiKohaku) [reply jekyll/_posts/2021-08-26-ospp-mid-term-wechaty-piggy-bro.md]: I removed this line in 5a0463a

**Review by Valerieeeee — APPROVED** [review]
LGTM

**Review by ninetailskim — APPROVED** [review]
LGTM

**Pull request merged**